src/experiments.ts

```diff
@@ -0,0 +1,2 @@
+// Components
+export * from './ActionList2'
```

Comment:
I like the idea of keeping experiments out of the main bundle. A couple of questions:

* If we want to build experimental components in primer, should we update the definition of "experimental" in this document? https://primer.style/contribute/component-lifecycle#experimental (defines "experimental" as built outside of Primer)
* Should we rename this file `experimental` to align with that document? ☝️ 
* When we release breaking changes to experimental components, which semver number should we bump?

Reply:
Hmm, there's a name conflict here for sure.

In our component lifecycle maturity, components with experimental maturity can still be in the main bundle.

What I want to signal here is - this is a "unreleased" component that is not part of the public api yet (not in main bundle). We don't recommend using it in production yet. You can still import it with explicitly for experimentation/feedback.

alternate names: unreleased, next, future, private, internal

Reply:
Update: changed to `unreleased`.

(`next` was a close second, but that might get confused with npm tag `primer/components@next`. `future` is romantic, but i'd like to put long running things in this scope that might not make it to main bundle at all)

@pksjce thoughts for new Button?

Reply:
(Useful to add, also added to PR description)

Why is ActionList2 in `unreleased` instead of replacing the current ActionList with a breaking change?

While I'm confident about the API in this PR (by testing it A LOT 😇), I'd like to "release" these changes along with their composable counterparts ActionMenu and SelectPanel, so that the memex team can catch up with breaking API for Menus together. This also gives us a change to tweak and polish the underlying ActionList to be accessible.